File: C Language/Assignments/Assi 3/palindrome.c
// Check the given number is palindrome or not? 
#include<stdio.h>
void main(){
    int n,temp;
    printf("Enter the number");
    scanf("%d",&n);
    int rev=0;
    temp=n; 
    while(n>0){
        int rem=n%10;
        rev=rev*10+rem;
        n=n/10;
    }
    if(rev==temp){
        printf("Palindrome");
    }
    else{
        printf("not palindrome");
    }
}

File: Java work/C to java Converted code/Assignment3/palindrome.java
package Assi3;

public class palindrome {
	public static void main() {
		int n=121;
		int temp;
	    int rev=0;
	    temp=n; 
	    while(n>0){
	        int rem=n%10;
	        rev=rev*10+rem;
	        n=n/10;
	    }
	    if(rev==temp){
	        System.out.println("Palindrome");
	    }
	    else{
	        System.out.println("not palindrome");
	    }
		
	}
	
}


